.moban.yaml

```diff
@@ -27,14 +28,22 @@ configuration:
     - .moban.dt/
     - ../coala-mobans/templates/
     - ../coala-mobans/assets/
-    - ../setupmobans/templates/
   configuration: .moban.yaml
   configuration_dir: ../coala-mobans/
 
+copy:
+  - docs-requirements.txt: docs/docs-requirements.txt
+  - docs/_static/images/coala_logo.svg: docs/images/coala_logo.svg
+  - docs/_static/images/coala-header.png: docs/images/coala-header.png
+  - docs/.gitignore: docs/.gitignore
+  - docs/Makefile: docs/Makefile
+  - docs/make.bat: docs/make.bat
+  - docs/_static/custom.css: docs/custom.css
+  - .circleci/run_with_env.cmd: run_with_env.cmd
```

Comment:
eh?? This is an AppVeyor file

Reply:
Yeh. Need to rename .circleci => .ci; Sorry for the confusion.